airbyte-server/src/test/java/io/airbyte/server/handlers/ArchiveHandlerTest.java

```diff
@@ -208,18 +211,47 @@ void testFullExportImportRoundTrip() throws Exception {
         .withNotifications(Collections.singletonList(notification))
         .withFirstCompletedSync(true)
         .withFeedbackDone(true);
-    final SourceConnection sourceConnection = new SourceConnection()
+    final SourceConnection source = new SourceConnection()
         .withSourceDefinitionId(sourceS3DefinitionId)
         .withSourceId(UUID.randomUUID())
-        .withWorkspaceId(standardWorkspace.getWorkspaceId())
+        .withWorkspaceId(workspace.getWorkspaceId())
         .withName("Test source")
         .withConfiguration(Jsons.deserialize("{}"))
         .withTombstone(false);
 
+    final StandardDestinationDefinition DESTINATION_S3 = new StandardDestinationDefinition()
```

Comment:
@alafanechere how did you test this works? I buffed up this test case to try and reproduce this. However, this is passing regardless of serialisation order in DatabaseConfigPersistence.java

Reply:
figure this out!